README.md

```diff
@@ -1,30 +1,33 @@
 About
 ------
-This repository is a fork of the work done in https://bitbucket.org/introom/docker-b.p.o
-
-This repository serves as a helper for running development instance of
-[roundup](http://roundup-tracker.org/) tracker living at http://bugs.python.org
+This repository allows you to create a Docker image able to run a development
+instance of the [Roundup](http://roundup-tracker.org/) tracker living at
+http://bugs.python.org
 
 
 Usage
-------
-
-1. Pull the docker image or [build](#build) one yourself:
+-----
+1. Clone the repo and build the image using:
 
   ```
-  sudo docker pull introom/b.p.o
+  make USERNAME=username
   ```
 
-  In the latter case use appropriate image naming in the following commands.
+  where `username` will be the repository name for your Docker image
+  (default: `unknown`).  You can also use e.g. `BASE_DISTRO=fedora` 
+  to specify the base distribution (default: `ubuntu`).
+
+  Note: in the future this image will be moved do DockerHub.
 
-2. Clone roundup and switch to branch `bugs.python.org`:
+2. Clone roundup and switch to the `bugs.python.org` branch:
 
   ```
   hg clone https://hg.python.org/tracker/roundup
-  hg checkout bugs.python.org
+  cd roundup
+  hg up bugs.python.org
```

Comment:
I like aliases myself, but for these types of guides, I prefer showing full commands.